candid/evm_rpc.did

```diff
@@ -178,7 +179,11 @@ type SendRawTransactionResult = variant {
 };
 type RequestResult = variant { Ok : text; Err : RpcError };
 type RequestCostResult = variant { Ok : nat; Err : RpcError };
-type RpcConfig = record { responseSizeEstimate : opt nat64 };
+type RpcConfig = record { responseSizeEstimate : opt nat64; responseConsensus : opt ConsensusStrategy };
+type ConsensusStrategy = variant {
+  Equality;
+  Threshold : record { num_providers : opt nat; min_num_ok : nat };
```

Comment:
`min_num_ok` sounds like we are happy if this number of `Ok` responses is received but the responses may actually be different, right? Don't we need equality among them?

Reply:
> Don't we need equality among them?

Yes we basically need `min_num_ok` consistent Ok responses. So how about renaming it to something like `min_num_consistent_ok`? Do you have any shorter alternatives 😅?

Reply:
Or just `min_num_consistent`?
I would say `consistent` implies `ok`.   :slightly_smiling_face: 
What do you think?

Reply:
`min_num_consistent` sounds good!. If that's ok with you, I would do the renaming in the follow-up PR (#287) to avoid git conflicts.

Reply:
@rvanasa : should the new field names maybe also use camel case as the other ones?

Reply:
> should the new field names maybe also use camel case as the other ones?

Yes, I think we should use camelCase for all of the field names for consistency.

Reply:
I like the idea. Nice and short.   :slightly_smiling_face:

Reply:
I also like the suggestion `total` and `min`, thanks @rvanasa for it! Since this PR was already approved, I will do the renaming in the next one to avoid conflicts.